doc-surrealdb_versioned_docs/version-latest/querying/index.mdx

```diff
@@ -0,0 +1,182 @@
+
+Now, let's explore the different methods to query data in SurrealDB.
+
+<ThemedImage
+  alt="QL"
+  width="70"
+  className="sdk-logo"
+  sources={{
+    light: useBaseUrl('img/light/ql.png'),
+    dark: useBaseUrl('img/ql-icon.png'),
+  }}
+/>
+
+## SurrealQL
+
+<br/>
+[SurrealQL](/docs/surrealql) is our powerful database query language that closely resembles traditional SQL but comes with unique differences and improvements.
+
+Designed to provide developers with a seamless and intuitive way to interact with SurrealDB, SurrealQL offers a familiar syntax and supports various statement types, allowing you to perform complex database operations efficiently.
+
+To get started with SurrealQL, explore the [SurrealQL documentation](/docs/surrealql) to learn about the [statements](/docs/surrealql/statements) available. These statements enable you to perform a wide range of database operations, from querying data to modifying records and managing database structures.
+
+Additionally, SurrealQL provides a rich set of [database functions](/docs/surrealql/functions/database) that enhance its capabilities. These functions allow you to perform advanced operations and leverage the full potential of SurrealDB's features. Whether you are working with data retrieval, manipulation, or complex computations, SurrealQL's functions offer the tools you need to build robust and scalable applications.
+
+
+### Getting Started
+
+The easiest way to get started with SurrealQL is to use [Surrealist](https://surrealist.app/). This interactive environment allows you to experiment with SurrealQL statements and see the results immediately. You can either set up a sandbox instance, a remote or local instance.
+
+
+
+You can use Surrealist to learn about the syntax and features of SurrealQL and to develop your queries and scripts.
+
+To familiarize yourself with SurrealQL, explore the various [statement](/docs/surrealql/statements) and their syntax. The statements pages provides comprehensive examples and explanations for each statement type, helping you understand how to construct queries and interact with SurrealDB effectively.
+
+SurrealQL empowers you to leverage the full potential of SurrealDB and enables you to build robust and scalable applications. Let's dive into the world of SurrealQL and unlock the capabilities of SurrealDB together!
+
+### Querying Options
+
+When using SurrealQL, there are several options available to interact with your database instance depending on your environment and needs. 
+
+- **Surrealist (Recommended)**: [Surrealist](https://surrealist.app/) is an interactive environment that allows you to experiment with SurrealQL statements and see the results immediately. It is a great tool for learning about the syntax and features of SurrealQL and for developing your queries and scripts.
```

Comment:
Both here and a paragraph or two above I see that Surrealist is introduced as the method that gives you results "immediately", which on its own would be fine but then contrasted with the other options below sort of makes it feel  like the others don't have that immediacy. So maybe a change to something else to emphasize how easy the results are to work with (graphically or somesuch, or emphasizing that they are in a separate view that is part of an overall control panel-like setup, not sure what the exact wording should be)